src/main/java/org/altbeacon/beacon/logging/LogManager.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2015 Radius Networks, Inc.
+ * Copyright 2015 Andrew Reitz
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.altbeacon.beacon.logging;
+
+/**
+ * Manager for logging in the Altbeacon library. The default is a
+ * {@link org.altbeacon.beacon.logging.Loggers#debug()}.
+ *
+ * @author Andrew Reitz
+ * @since 2.2
+ */
+public final class LogManager {
+    private static Logger sLogger = Loggers.debug();
```

Comment:
By default, it appears this will use the debug logger, which will output everything (debug, info, warning, error) to LogCat, correct?  I think the default should be that it only outputs only warning and error messages.